core/templates/pages/exploration-editor-page/services/setting-tab-backend-api.service.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Backend api service for Setting tab;
+ */
+
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+interface SettingTabBackendDict {
+  'draftEmailBody': string;
+}
+
+interface SettingTabResponse {
+  draftEmailBody: string
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SettingTabBackendApiService {
+  constructor(
+    private http: HttpClient
+  ) {}
+
+  getData(url: string): Promise<SettingTabResponse> {
+    return this.http.get<SettingTabBackendDict>(
+      url
+    ).toPromise();
```

Comment:
Can't this all be in one line?
```suggestion
    return this.http.get<SettingTabBackendDict>(url).toPromise();
```

Reply:
Done